src/sentry/api/endpoints/group_events.py

```diff
@@ -49,18 +54,23 @@ def get(self, request, group):
         """
 
         try:
-            environment = self._get_environment(request, group)
-            query, tags = self._get_search_query_and_tags(request, group, environment)
+            environments = get_environments(request, group.project.organization)
+            query, tags = self._get_search_query_and_tags(
+                request,
+                group,
+                environments,
+            )
         except InvalidQuery as exc:
             return Response({'detail': six.text_type(exc)}, status=400)
-        except NoResults:
+        except (NoResults, ResourceDoesNotExist):
             return Response([])
 
         use_snuba = options.get('snuba.events-queries.enabled')
```

Comment:
It's disabled at the moment, I'll add a forcing param in.

Reply:
can I -1 this, or -0.5 it maybe. 

There are already quite a few ways to do backend selection. There are cookies, these option flags, the primary/secondary endpoint selection logic, and places where different backends are hardcoded. I'm a little wary of adding another way. 

IMHO we should probably just hardcode the snuba backend once we want to use this live, admittedly this depends on me fixing my stuff so that the snuba result works with the current group events tab, so I can try and get on that soon.

Or would that approach needlessly block you guys from working on the new issue details stuff as you have no way to control the endpoint in use from the UI code?

Reply:
I guess maybe add a URL param, as long as we are going to remove it once we go live with the new issue details page.

Reply:
I don't feel strongly either way. I thought it was just a temporary param so that we can test until we enable the feature switch, but I'm not even really sure how we intend to use it. Thoughts, @macqueen?

Reply:
yeah temp param fine by me

Reply:
yeah sorry i think i was mis-remembering how selection was done in this diff and thought it was a url param and not a cookie: https://github.com/getsentry/sentry/pull/10097 (which i think has now been refactored / removed now anyway)

Whatever we do, yes, agreed it would just be a temp override until `options.get('snuba.events-queries.enabled')` is `True` for sentry.io and then it should be removed.

Reply:
Let's just leave it as a temporary querystring param, and I'll remove it once we're using the Snuba stuff entirely?